PantheonCMD/pcbuild.py

```diff
@@ -53,6 +55,20 @@ def build_content(content_files, lang, repo_location, yaml_file_location):
     return True
 
 
+def parse_attributes(attributes):
```

Comment:
If I u sweat and correctly, here were parsing attributes from the main attributes file. In rhel repo for example we also have local attributes per every title + some teams identify their own attributes in the file itself. While local attributes usually don’t contain any path attributes in my experience, I’ve seen people defining attributes in source files that they use as part of the path.

Reply:
Great point, @Levi-Leah - I have now updated 'coalesce_document' so that it creates a copy of the attributes dictionary, then updates the original with each successive include so that it contains a reference of all found attributes. If there are two attributes with the same key, the value will get overwritten, but this should be fine, because it's processing includes and attributes as it finds them, and the value of any attributes needs to be correct as of the time it's needed.

At the end of coalesce_document, we then use the copy as a set of only the global attributes, and write them to the main file so that we're not accidentally pulling in anything that needs to be declared in a specific place in the file. Those will stay where they are.